contracts/implementations/nodes/AftermarketDevice.sol

```diff
@@ -35,6 +38,8 @@ contract AftermarketDevice is
         keccak256(
             "UnPairAftermarketDeviceSign(uint256 aftermarketDeviceNode,uint256 vehicleNode)"
         );
+    uint256 private constant MANUFACTURER_MINTER_PRIVILEGE = 1;
```

Comment:
Do you think maybe these should be configurable?

Reply:
How configurable?